platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/ProgramExecutionRunnable.java

```diff
@@ -70,17 +81,22 @@ private AGProgramExecution updateStartedAt() {
 
 	private void executeProgram() {
 
-		try (LogOutputScope scope = new LogOutputScope(logBuffer)) {
-			new ProgramStarter(getProgramUuid()).start();
-		} catch (Throwable throwable) {
-			this.failed = true;
-			logStackTrace(throwable);
+		try (var logFileOutput = new LogFileOutput(logFilePath)) {
+			var logOutputDistributor = new MultiLogOutput(logFileOutput, logBuffer);
+			try (var scope = new LogOutputScope(logOutputDistributor)) {
```

Comment:
either
```suggestion
			try (var scope = new LogOutputScope(new MultiLogOutput(logFileOutput, logBuffer))) {
```
or
```suggestion
			var logOutput = new MultiLogOutput(logFileOutput, logBuffer);
			try (var scope = new LogOutputScope(logOutput)) {
```

Reply:
can't rename a variable, I though `logOutputDistributor` was just a leftover from mutliple renamings of the class

Reply:
Oh... nvm, I misread the lower suggestion.